src/main/java/org/apache/sling/jcr/registration/impl/JndiRegistrationSupport.java

```diff
@@ -53,20 +55,17 @@
 @Component(
         configurationPolicy = ConfigurationPolicy.REQUIRE,
         name = "org.apache.sling.jcr.jackrabbit.server.JndiRegistrationSupport",
-        property = {
-                "service.vendor=The Apache Software Foundation",
-                "service.description=JNDI Repository Registration"
-        },
         reference = {
                 @Reference(name = "Repository", policy = ReferencePolicy.DYNAMIC,
-                        bind = "bindRepository", unbind = "unbindRepository",
                         cardinality = ReferenceCardinality.MULTIPLE, service = Repository.class),
                 @Reference(name = "Log", policy = ReferencePolicy.DYNAMIC,
                         bind = "bindLog", unbind = "unbindLog",
                         cardinality = ReferenceCardinality.OPTIONAL, service = LogService.class),
         }
 )
 @Designate(ocd = JndiRegistrationSupport.Configuration.class)
+@ServiceVendor("The Apache Software Foundation")
```

Comment:
We agreed that this is superfluous as the bundle vendor is already set (which implicitly applies to all contained services)